assets/base.json

```diff
@@ -0,0 +1,6 @@
+{
+    "description": "Women In Web 3 - Credential Token",
```

Comment:
**WOMEN IN WEB3** across all pages (everywhere) pls!

Reply:
Got it!